retrieval_service/datastore/providers/cloudsql_postgres.py

```diff
@@ -422,5 +422,25 @@ async def search_flights_by_airports(
         res = [models.Flight.model_validate(r) for r in results]
         return res
 
+    async def insert_ticket(
+        self,
+        user_id: str,
+        user_name: str,
+        user_email: str,
+        airline: str,
+        flight_number: str,
+        departure_airport: str,
+        arrival_airport: str,
+        departure_time: str,
+        arrival_time: str,
+    ):
+        raise NotImplementedError("Not Implemented")
```

Comment:
Any reason we aren't implementing here?

Reply:
Will implement cloudsql and firestore endpoints in separate PRs